### PR TITLE
Add changelog for v1.11.1

### PR DESCRIPTION
**Pull request by mlaventure**
Signed-off-by: Kenfe-Mickael Laventure mickael.laventure@gmail.com

---

**NOTE**: This include entries for PR which have not been merged yet

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,40 @@ information on the list of deprecated flags and APIs please have a look at
...
+* Append label passed to `docker build` as arguments as an implicit `LABEL` command at the end of the processed `Dockerfile` ([#22184](https://github.com/docker/docker/pull/22184))
+
+### Networking
+
+- Fix a panic that would occur when forwarding DNS query ([#22261](https://github.com/docker/docker/pull/22261))
+- Fix a panic that would occur when the network plugins were activated via the json spec ([#22191](https://github.com/docker/docker/pull/22191))
+
+### Runtime
+
+- Fix a regression where container mountaing `/var/run` would prevent other containers from being removed ([#22256](https://github.com/docker/docker/pull/22256))
```
Comment (thaJeztah) [inline CHANGELOG.md]: s/mountaing/mounting/

**Comment by mlaventure** [comment]
Updated with the latest PRs added to `1.11.1`.

PTAL.

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,43 @@ information on the list of deprecated flags and APIs please have a look at
...
+* Append label passed to `docker build` as arguments as an implicit `LABEL` command at the end of the processed `Dockerfile` ([#22184](https://github.com/docker/docker/pull/22184))
+
+### Networking
+
+- Fix a panic that would occur when forwarding DNS query ([#22261](https://github.com/docker/docker/pull/22261))
+- Fix a panic that would occur when the network plugins were activated via the json spec ([#22191](https://github.com/docker/docker/pull/22191))
+
+### Runtime
+
+- Fix a bug preventing labels configuration to be reloade via the config file ([#22299](https://github.com/docker/docker/pull/22299))
```
Comment (thaJeztah) [inline CHANGELOG.md]: s/reloade/reloaded/

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,43 @@ information on the list of deprecated flags and APIs please have a look at
...
+- Fix the handling of symlinks when passed to `--devices` ([#22272](https://github.com/docker/docker/pull/22272))
```
Comment (thaJeztah) [inline CHANGELOG.md]: relative symlinks (?)

File: CHANGELOG.md [hunk 4]
```diff
@@ -5,6 +5,43 @@ information on the list of deprecated flags and APIs please have a look at
...
++ Add missing API documentation for changes introduced with 1.11.0 ([#22048](https://github.com/docker/docker/pull/22048))
+
+### Builder
+
+* Append label passed to `docker build` as arguments as an implicit `LABEL` command at the end of the processed `Dockerfile` ([#22184](https://github.com/docker/docker/pull/22184))
+
+### Networking
+
+- Fix a panic that would occur when forwarding DNS query ([#22261](https://github.com/docker/docker/pull/22261))
+- Fix a panic that would occur when the network plugins were activated via the json spec ([#22191](https://github.com/docker/docker/pull/22191))
```
Comment (mavenugo) [inline CHANGELOG.md]: It is not specific to network plugins. The issue is in the plugins processing logic that is applicable for all the plugin types.

Comment (mlaventure) [inline CHANGELOG.md]: Thanks, will update

File: CHANGELOG.md [hunk 5]
```diff
@@ -5,6 +5,43 @@ information on the list of deprecated flags and APIs please have a look at
...
+
```
Comment (sanimej) [inline CHANGELOG.md]: Fix the issue of OS threads stuck in incorrect network namespace ([#22261](https://github.com/docker/docker/pull/22261))

File: CHANGELOG.md (hunk 4 above)
Comment (mavenugo) [inline CHANGELOG.md]: thanks. also move it to Runtime.

**Comment by mlaventure** [comment]
Updated as 3 PRs were removed from the milestone

File: CHANGELOG.md [hunk 6]
```diff
@@ -5,6 +5,40 @@ information on the list of deprecated flags and APIs please have a look at
...
+* Append label passed to `docker build` as arguments as an implicit `LABEL` command at the end of the processed `Dockerfile` ([#22184](https://github.com/docker/docker/pull/22184))
+
+### Networking
+
+- Fix a panic that would occur when forwarding DNS query ([#22261](https://github.com/docker/docker/pull/22261))
+
+### Runtime
+
+- Fix a bug preventing labels configuration to be reloade via the config file ([#22299](https://github.com/docker/docker/pull/22299))
```
Comment (thaJeztah) [inline CHANGELOG.md]: s/reloade/reloaded/ 😇

**Comment by thaJeztah** [comment]
one minor nit, but LGTM otherwise

File: CHANGELOG.md [hunk 7]
```diff
@@ -5,6 +5,40 @@ information on the list of deprecated flags and APIs please have a look at
...
++ Add missing API documentation for changes introduced with 1.11.0 ([#22048](https://github.com/docker/docker/pull/22048))
+
+### Builder
+
+* Append label passed to `docker build` as arguments as an implicit `LABEL` command at the end of the processed `Dockerfile` ([#22184](https://github.com/docker/docker/pull/22184))
+
+### Networking
+
+- Fix a panic that would occur when forwarding DNS query ([#22261](https://github.com/docker/docker/pull/22261))
+
```
Comment (sanimej) [inline CHANGELOG.md]: To be added under Networking..

Fix the issue of OS threads stuck in incorrect network namespace when using user defined networks (#22261)

**Comment by mlaventure** [comment]
@thaJeztah updated

**Comment by tiborvass** [comment]
LGTM

**Pull request merged**